__tests__/src/config.test.ts

```diff
@@ -55,7 +55,7 @@ describe("Given a configuration", () => {
 
     const config = configFactory.getConfiguration()
 
-    expect(config.shaArchive).toBe("https://api.github.com/repos/mpermar/vib-action-test/tarball/a-new-branch")
+    expect(config.shaArchive).toBe("https%3A%2F%2Fapi.github.com%2Frepos%2Fmpermar%2Fvib-action-test%2Ftarball%2Fa-new-branch")
```

Comment:
The original issue is only referring to URL containing `#` hashes. Why are we changing all the other characters? Does that URL work?

Reply:
What would occur if there are multiple hashes? 

For example, `https://github.com/bitnami/charts/tree/feat/%2312580-add-cronjob-for-matomo-chart#tldr` is a valid URL but `https://github.com/bitnami/charts/tree/feat/%2312580-add-cronjob-for-matomo-chart%23tldr` I think it is not.